### PR TITLE
PLT-3073: Implement SAML/Okta Server side (EE)

### PR DESCRIPTION
**Pull request by enahum**

### CODE REVIEW

File: config/config.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
         "MaxUsersPerTeam": 50,
         "EnableTeamCreation": true,
         "EnableUserCreation": true,
-        "EnableOpenServer": false,
```
Comment (jwilander) [inline config/config.json]: This is a mistake right? I don't think we want to change the default people use for this setting

File: glide.lock [hunk 2]
```diff
@@ -61,6 +63,8 @@ imports:
   - i18n/bundle
   - i18n/language
   - i18n/translation
+- name: github.com/nu7hatch/gouuid
```
Comment (jwilander) [inline glide.lock]: What's this for? I see it being used in the enterprise PR but is there a reason we can't use the guid generator we already have?

File: model/config.go [hunk 3]
```diff
@@ -220,6 +220,28 @@ type LocalizationSettings struct {
...
+	IdpUrl                      *string
+	IdpDescriptorUrl            *string
+	LoginButtonText             *string
+	AssertionConsumerServiceURL *string
+
+	// User Mapping
+	FirstNameAttribute *string
+	LastNameAttribute  *string
+	EmailAttribute     *string
+	UsernameAttribute  *string
```
Comment (jwilander) [inline model/config.go]: Missing a default for this one

File: config/config.json (hunk 1 above)
Comment (enahum) [inline config/config.json]: yeap my bad, I'll fix this

**Comment by enahum** [comment]
@jwilander done with the changes

**Comment by jwilander** [comment]
+1

**Comment by jasonblais** [comment]
+1 for PM approval

**Comment by crspeller** [comment]
+1

**Pull request merged**